src/OpenTelemetry.AutoInstrumentation/Configurations/LogSettings.cs

```diff
@@ -18,9 +18,9 @@ internal class LogSettings : Settings
     public bool LogsEnabled { get; private set; }
 
     /// <summary>
-    /// Gets the logs exporter.
+    /// Gets the list of enabled logs exporters.
     /// </summary>
-    public LogExporter LogExporter { get; private set; }
+    public IReadOnlyList<LogExporter> LogExporters { get; private set; } = new List<LogExporter>();
```

Comment:
Given that the property is set on `OnLoad` do we really need to create an instance here? Similar Q for the other signals.

Reply:
At the moment, this needs to remain as is to avoid nullable issues. However, we can refactor this in a separate PR in the future.